assets/js/bespoke.js

```diff
@@ -0,0 +1,9 @@
+window.onclick = function () {
+  var $addAnothers = document.querySelectorAll('[data-module="moj-add-another"]')
+  const addAnother = new MOJFrontend.AddAnother($addAnothers)
+  const buttons = document.getElementsByClassName('delete-condition-button')
+  if (buttons.length) {
```

Comment:
do you need to do this check ? can you not just rely on `getItems` here?

Reply:
Yes, we need it.
It is not remove button count, it is to get the delete condition button to rename the text content.